pyproject.toml

```diff
@@ -216,6 +217,8 @@ required_plugins = [
 addopts = [
     "-nauto",
     "-vra",
+    "-ra",
+    "--showlocals",
```

Comment:
I'm curious: is there a reason you think we should opt for this option? Do you feel that our tracebacks are limited?

Reply:
This conversation is still relevant, as I'm not sure why we need to include this option with this PR – is it related to the changes? It's useful, but if it doesn't need to be included for the main changes here, please consider adding it in another PR.